### DISCUSSION TITLE
How to include table from tables from multiple database

### DISCUSSION THREAD

**Discussion by sbisen** [vanna-ai/vanna#366] [Q&A]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by sbisen** [comment]
These multiple DB are in one snowflake account

**Comment by zainhoda** [comment]
@sbisen as long as you have the relevant training data, this should just work out of the box. Are you facing some specific issues? What questions are you asking and what training data do you have?
